### DISCUSSION TITLE
Does this library support BatchGet on multiple table?

### DISCUSSION THREAD

**Discussion by kunalfleethawks** [sensedeep/dynamodb-onetable#246] [Q&A] [answered]
As per the design of this library which is supposed to handle a single table and the reason BatchGet is only enabled on a single table. Is there a way to support BatchGet on multiple tables? As AWS DynamoDB supports it . 

https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchGetItem.html#API_BatchGetItem_Examples 

> The BatchGetItem operation returns the attributes of one or more items from one or more tables. 

Let me know your thoughts on this. Thank you.

**Comment by mobsense** [comment, marked as answer]
Not at this stage. You can create multiple OneTable Table() instances and manage interactions with multiple tables, but the batch* and transact* APIs currently only work with one table at a time.
